applications/doodle/src/doodle_exe.erl

```diff
@@ -253,7 +253,7 @@ init([Call]) ->
     process_flag('trap_exit', 'true'),
     CallId = whapps_call:call_id(Call),
     put('callid', CallId),
-    self() ! 'initialize',
+    gen_listener:cast(self(), 'initialize'),
```

Comment:
Ie, would it be a valid PR if I changed the `handle_info('initialize', #state{call=Call})` to be `handle_cast('initialize', #state{call=Call})` ?